code/src/WijmoProvider/Columns/AbstractProviderColumn.ts

```diff
@@ -149,7 +152,8 @@ namespace WijmoProvider.Column {
             //RGRIDT-574 review after solved
             //Error when the column is inside a ColumnGroup
             !this.hasParentColumn &&
-                (this.grid.provider as wijmo.grid.FlexGrid).columns.remove(
+                //@ts-ignore
```

Comment:
why do we need this ts-ignore?

Reply:
Basically, because wijmo defined column groups as columnGroups: any[], even though they extend ObservableArray. So, since its any, it does not recognize the insert method.